### DISCUSSION TITLE
Looking for a complete Project Walkthrough

### DISCUSSION THREAD

**Discussion by andjosaus** [asreview/asreview#1328] [Q&A]
Hi!

I came across ASReview Lab today and was trying to use it but had a couple issues when it came to setting up a new project. 

- What is the best way to organize the papers I want to analyze into a file I can upload?
- Any help understanding the data would also be beneficial.
- Is there a way I can manually feed the platform articles (one-by-one) using urls?

A full instructional video would be lovely to really help grasp how to use the platform beyond the current Youtube Videos and promotional videos..

Thanks

**Comment by Rensvandeschoot** [comment]
You might want to visit the [YouTube channel](https://www.youtube.com/@ASReview/videos)  of ASReview

**Comment by andjosaus** [comment]
I have gone through those videos, but was looking for more information as I mentioned above

**Reply by Rensvandeschoot** [reply]
Ok! let me give it another try:

- How to prepare the dataset is explained in [a section of the documentation](https://asreview.readthedocs.io/en/latest/data.html). It should contain text to be screened, for example, the meta-data of scientific papers (i.e., titles and abstract), but any textual data can be used (e.g., newspaper articles, emails, etc.) as explained in[ this part of the documentation](https://asreview.readthedocs.io/en/latest/data_format.html). 
- Data pre-processing and obtaining descriptive statistics can be done via [Datatools](https://github.com/asreview/asreview-datatools).
- Feeding data one by one is not a feature we currently support, but this might be possible via the API (@J535D165 ?), but typically users first collect a dataset they want to screen and then import this dataset in ASReview, see the [Get Started walkthrough](https://asreview.readthedocs.io/en/latest/about.html) in the documentation.
